populate_db.py

```diff
@@ -49,24 +55,64 @@ def create_permissions():
     # For ORGANIZER
-    get_or_create(Permission, role=orgr, service=track, can_create=True, can_read=True, can_update=True, can_delete=True)
-    get_or_create(Permission, role=orgr, service=session, can_create=True, can_read=True, can_update=True, can_delete=True)
-    get_or_create(Permission, role=orgr, service=speaker, can_create=True, can_read=True, can_update=True, can_delete=True)
-    get_or_create(Permission, role=orgr, service=sponsor, can_create=True, can_read=True, can_update=True, can_delete=True)
-    get_or_create(Permission, role=orgr, service=microlocation, can_create=True, can_read=True, can_update=True, can_delete=True)
+    # All four permissions set to True
+    get_or_create(Permission, role=orgr, service=track)
+    get_or_create(Permission, role=orgr, service=session)
+    get_or_create(Permission, role=orgr, service=speaker)
+    get_or_create(Permission, role=orgr, service=sponsor)
+    get_or_create(Permission, role=orgr, service=microlocation)
 
     # For COORGANIZER
-    get_or_create(Permission, role=coorgr, service=track, can_create=False, can_read=True, can_update=True, can_delete=False)
-    get_or_create(Permission, role=coorgr, service=session, can_create=False, can_read=True, can_update=True, can_delete=False)
-    get_or_create(Permission, role=coorgr, service=speaker, can_create=False, can_read=True, can_update=True, can_delete=False)
-    get_or_create(Permission, role=coorgr, service=sponsor, can_create=False, can_read=True, can_update=True, can_delete=False)
-    get_or_create(Permission, role=coorgr, service=microlocation, can_create=False, can_read=True, can_update=True, can_delete=False)
+    perm, _ = get_or_create(Permission, role=coorgr, service=track)
+    perm.can_create, perm.can_delete = False, False
+    db.session.add(perm)
+
+    perm, _ = get_or_create(Permission, role=coorgr, service=session)
+    perm.can_create, perm.can_delete = False, False
+    db.session.add(perm)
+
+    perm, _ = get_or_create(Permission, role=coorgr, service=speaker)
+    perm.can_create, perm.can_delete = False, False
+    db.session.add(perm)
+
+    perm, _ = get_or_create(Permission, role=coorgr, service=sponsor)
+    perm.can_create, perm.can_delete = False, False
+    db.session.add(perm)
+
+    perm, _ = get_or_create(Permission, role=coorgr, service=microlocation)
+    perm.can_create, perm.can_delete = False, False
+    db.session.add(perm)
 
     # For TRACK_ORGANIZER
-    get_or_create(Permission, role=track_orgr, service=track, can_create=False, can_read=True, can_update=True, can_delete=False)
+    perm, _ = get_or_create(Permission, role=track_orgr, service=track)
+    db.session.add(perm)
 
     # For MODERATOR
-    get_or_create(Permission, role=mod, service=track, can_create=False, can_read=True, can_update=False, can_delete=False)
+    perm, _ = get_or_create(Permission, role=mod, service=track)
+    perm.can_create, perm.can_update, perm.can_delete = False, False, False
+    db.session.add(perm)
```

Comment:
No, db is in the global namespace. But I've moved it inside `populate()` so it's clear the session is committed after all the functions have been called.